.github/workflows/main.yml

```diff
@@ -7,9 +10,21 @@ on:
     branches: [develop]
 
 jobs:
-  build:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: psf/black@20.8b1
+        with:
+          args: ". --check"
 
+  build:
+    needs: lint
     runs-on: Ubuntu-20.04
+    strategy:
+      matrix:
+        python: [3.7, 3.9]
```

Comment:
Why 3.6 and 3.8 are not here?

Reply:
> Why 3.6 and 3.8 are not here?

3.9 because it's the latest and 3.7 because that's what isabel started developing on I guess not sure cc @isabelcosta

Reply:
@epicadk is right. @devkapilbansal do you think having those versions here would help in any way? if yes, then I'm okay with adding those

Reply:
> If we are keeping 3.7 as lowest supported version, then probably we should add all versions after that. Just to make sure nothing breaks.

Makes sense for a library. But for an end user application I don't think that's required : /.